apps/discovery-platform/src/db/index.spec.ts

```diff
@@ -100,7 +100,9 @@ describe("test db functions", function () {
     await db.saveRegisteredNode(dbInstance, secondNode);
     await db.saveRegisteredNode(dbInstance, createMockNode());
 
-    const notExitNodes = await db.getNonExitNodes(dbInstance);
+    const notExitNodes = await db.getRegisteredNodesWithFilters(dbInstance, {
```

Comment:
why not turn `getRegisteredNodesWithFilters` to `getRegisteredNodes` and make the filters optional argument?